micromamba/tests/test_remove.py

```diff
@@ -64,7 +61,7 @@ def test_remove_orphaned(tmp_home, tmp_root_prefix, tmp_xtensor_env, tmp_env_nam
     # assert len(res["actions"]["UNLINK"]) == len(env_pkgs) + (
     assert len(res["actions"]["UNLINK"]) == 3 + (
         1 if helpers.dry_run_tests == helpers.DryRun.DRY else 0
-    )
+    ) + (platform.system() == "Linux")  # xtl is not removed on Linux
```

Comment:
Can we remove this and try with adding back the commented line? `len(env_pkgs)` instead of `3`.